src/server/apsystem.ts

```diff
@@ -294,10 +294,26 @@ export default class ActivityPubSystem {
 
     const { type } = activity
 
-    // TODO: Handle other types + index by post
     if (type === 'Follow') {
       await this.acceptFollow(fromActor, activity)
+    } else if (type === 'Create') {
+      if ('object' in activity && activity.object !== undefined && typeof activity.object !== 'string') {
+        const activityObject = activity.object as APActivity
+        if (
+          activityObject.type === 'Note' &&
+          activityObject.inReplyTo !== '' &&
```

Comment:
```suggestion
          (typeof activityObject.inReplyTo === 'string') && activityObject.inReplyTo !== '' &&
```

we should make sure it's a string in case it's undefined or null or something